csync/src/vio/csync_vio_local.c

```diff
@@ -384,6 +389,10 @@ int csync_vio_local_stat(const char *uri, csync_vio_file_stat_t *buf) {
       buf->flags |= CSYNC_VIO_FILE_FLAGS_HIDDEN;
   }
 #endif
+  if( strlen(buf->name) > 1 && buf->name[0] == '.' ) {
```

Comment:
Is there ever a chance that this gets called with "." as current directory or ".." or so? i guess not?